release notes/upcoming.md

```diff
@@ -12,6 +12,7 @@ console.log(rnd)
 ```
 
 * A new option `--no-vu-connection-reuse` lets users close HTTP `keep-alive` connections between iterations of a VU. (#676)
+* New options were added to the HAR converter. You can set the minimum and maximum sleep time at the end of an iteration with the new `--min-sleep` and `--max-sleep` CLI flags of `k6 convert`. You can also specify a JSON file with [script options](https://docs.k6.io/docs/options) that would be added to the options of the generated scripts with the new `--options` flag. (#694)
```

Comment:
I'd separate the `--min-sleep`/`--max-sleep` and `--options` into two separate items.

Reply:
Done: https://github.com/loadimpact/k6/pull/694/commits/8660bad5b368d174ff90a138e19e5623b36398de